pkg/client/unversioned/helper.go

```diff
@@ -672,3 +709,18 @@ func TimeoutFromListOptions(options api.ListOptions) time.Duration {
 	}
 	return 0
 }
+
+func stripAPIFromPrefix(base string) string {
+	// normalize the base
```

Comment:
I meant with the trailing "/". Didn't follow all the control flow. Maybe it's resolved now.